### PR TITLE
for Blasr to work correctly with pbbam new handling of empty filters

### PR DESCRIPTION
**Pull request by vrainish-pacbio**

### CODE REVIEW

File: alignment/files/ReaderAgglomerate.cpp [hunk 1]
```diff
@@ -227,12 +227,7 @@ int ReaderAgglomerate::Initialize() {
                 assert(sequentialZmwQueryPtr != nullptr);
                 sequentialZmwIterator = sequentialZmwQueryPtr->begin();
             } else if (fileType == PBDATASET) {
-                    // It is necessary to construct a default filter which does not 
-                    // filter any records, because an empty filter throws away all records.
-                    PacBio::BAM::PbiFilter filter{ PacBio::BAM::PbiQueryLengthFilter{ 0 , PacBio::BAM::Compare::GREATER_THAN } };
-                if (dataSetPtr->Filters().Size() != 0) {
-                    filter = PacBio::BAM::PbiFilter::FromDataSet(*dataSetPtr);
-                }
+		const PacBio::BAM::PbiFilter filter = PacBio::BAM::PbiFilter::FromDataSet(*dataSetPtr);
```
Comment (pb-cdunn) [inline alignment/files/ReaderAgglomerate.cpp]: Try to fix the settings in your editor. Your indentation is wrong here. You're using tabs in a file which uses spaces.

**Comment by vrainish-pacbio** [comment]
Hi Chris,

Should I do something else in addition ?
I recall last time it was some additional action related to our coexistence with p4,
But things has changed since then ( I paid attention travis was launched automatically when I created pull request)

Regards, Vladimir

**Comment by pb-cdunn** [comment]
You have to update *.ini in bioinformatics/ext/pi/. Ask someone there. I'll be online again later tonight. This mnight help:

* https://github.com/PacificBiosciences/pb-git/wiki/Usage

**Comment by pb-cdunn** [comment]
I updated this in p4 for you, since I had another change behind it.

**Pull request merged**